Extension/src/LanguageServer/extension.ts

```diff
@@ -959,17 +960,64 @@ function reportMacCrashes(): void {
     }
 }
 
-let previousMacCrashData: string;
-let previousMacCrashCount: number = 0;
+export function watchForCrashes(crashDirectory: string): void {
+    if (process.platform !== "win32") {
+        prevCrashFile = "";
+        fs.stat(crashDirectory, (err) => {
+            const crashObject: Record<string, string> = {};
+            if (err?.code) {
+                // If the directory isn't there, we have a problem...
+                crashObject["fs.stat: err.code"] = err.code;
```

Comment:
I'd prefer to have only alpha characters in event property names.  Can you call it "errorCode" instead?
This appears to be the same way the old mac crashes were reported but I can attest that I never queried this property.